lib/utils/json.c

```diff
@@ -1218,6 +1218,10 @@ static int json_escape_internal(const char *str,
 	const char *cur;
 	int ret = 0;
 
+	if (!str) {
```

Comment:
No implicit boolean conversion please.

```suggestion
	if (str == NULL) {
```